pkg/analyzer/analyzers/asana/asana.go

```diff
@@ -102,3 +148,11 @@ func printMetadata(me *MeJSON) {
 	}
 	t.Render()
 }
+
+func allPermissions() []analyzers.Permission {
+	permissions := make([]analyzers.Permission, 1)
+	permissions = append(permissions, analyzers.Permission{
+		Value: "Full Access",
```

Comment:
Two options:
1. Use `analyzers.FullAccess` constant for the value here
2. Generate permissions / data types for just this one type of permission

I'm leaning towards 2 for consistency, even if it may be overkill.

Reply:
I would also lean toward option # 2.